spec/models/user_spec.rb

```diff
@@ -648,6 +648,19 @@
     expect( user ).to be_suspended
   end
 
+  describe "moderated_with" do
+    it "renames the user when given a RENAME action" do
+      user = User.make!( login: "old_login" )
+      moderator_action = ModeratorAction.new(
+        action: ModeratorAction::RENAME,
+        user: User.make!,
+        resource: user
+      )
```

Comment:
Not critical, but if you use the FactoryBot factory you can omit the user since it will create one by default, e.g.

```ruby
moderator_action = build :moderator_action, action: ModeratorAction::RENAME, resource: user
```